oshi-core/src/main/java/oshi/software/os/mac/MacInstalledApps.java

```diff
@@ -31,6 +36,7 @@ private static List<ApplicationInfo> parseMacAppInfo(List<String> lines) {
         String appName = null;
         Map<String, String> appDetails = null;
         boolean collectingAppDetails = false;
+        String dateFormat = getMacDateTimeFormat(CoreFoundation.kCFDateFormatterShortStyle);
```

Comment:
1. This shouldn't ever change. Let's make this a static class field.
2. Consider a static import for this.

Reply:
1. User could alter region on-the-fly in macOS...
2. ok!

Reply:
> 1. User could alter region on-the-fly in macOS...

Fair point.  I guess one call before the app loop isn't horrible.